### DISCUSSION TITLE
Can we have a separate awesome list for v2

### DISCUSSION THREAD

**Discussion by CxRes** [vuepress/core#593] [Q&A] [answered]
I am looking to port a site to Vuepress and was thinking of using v2. However, when I am finding that searching for themes and plugins is an absolute pain because it is unclear if they support v2, unless you navigate their documentation. It would be much nicer to have everything in one place!

**Comment by Mister-Hope** [comment, marked as answer]
Please check #735

**Comment by CxRes** [comment]
@Mister-Hope I checked the awesome page. There is no separation and/or clarity on which items work with v1 and which with v2. One has to check pages of individual components, and that is painful. :(

**Reply by Mister-Hope** [reply]
Like I said, I will start this these days. Just noticing you to track as there are several related dicussion in the dicussion panel.

**Reply by CxRes** [reply]
Ok. I thought "it will be ready **these** days" meant it was ready. Not a problem, thanks for looking at the issue!

**Reply by Mister-Hope** [reply]
"Will be" is "will be" bro 😎
